src/components/Filter/Filter.tsx

```diff
@@ -108,6 +117,20 @@ export const Filter: FC<FilterComponentProps> = ({
         setActiveSection(newSection);
     };
 
+    const updateFiltersWithPresets = (value: string) => {
+        if (filterMapping) {
+            const filtersToPreset = filterMapping[value];
+            Object.entries(filtersToPreset).forEach(([key, value]) => {
```

Comment:
nit: Even though scoping applies/works here, might read a little easier/avoid confusion if you rename one of the `value` argumements (either on line 120 or 123).

Reply:
oh shoot that's a good call, didn't realize i was using the same variable name.